webui/src/pages/repositories/repository/objects.jsx

```diff
@@ -291,6 +296,31 @@ const TreeContainer = ({
     );
 }
 
+const READMEContainer = ({repo, reference}) => {
+    const {response, error, loading} = useAPI(async () => {
+        return await objects.get(repo.id, reference.id, 'README.md');
+    });
+
+    if (loading || error) {
+        return <></>
```

Comment:
Ideally we would display any non-404 errors: if we got here and there _is_ a README.md, the user might be expecting to see it!